com.unity.render-pipelines.universal/Shaders/Utils/StencilDeferred.shader

```diff
@@ -209,13 +218,18 @@ Shader "Hidden/Universal Render Pipeline/StencilDeferred"
         half alpha = 1.0;
 
         #if defined(_SCREEN_SPACE_OCCLUSION)
-        AmbientOcclusionFactor aoFactor = GetScreenSpaceAmbientOcclusion(screen_uv);
-        unityLight.color *= aoFactor.directAmbientOcclusion;
+            AmbientOcclusionFactor aoFactor = GetScreenSpaceAmbientOcclusion(screen_uv);
+            unityLight.color *= aoFactor.directAmbientOcclusion;
             #if defined(_DIRECTIONAL) && defined(_DEFERRED_FIRST_LIGHT)
-            alpha = aoFactor.indirectAmbientOcclusion;
+            half occlusion = min(surfaceDataOcclusion, aoFactor.indirectAmbientOcclusion);
+            // What we want is really to apply the mininum occlusion between the baked occlusion from surfaceDataOcclusion and real-time occlusion from SSAO.
+            // But we already applied the baked occlusion during gbuffer pass, so we have to cancel it out here.
+            occlusion /= surfaceDataOcclusion;
```

Comment:
Thought: Would changing this to `occlusion *= rcp(surfaceDataOcclusion)` be faster than a division or make no difference at all?

Reply:
Ok, done!